lib/hammer_cli_foreman/attribute.rb

```diff
@@ -0,0 +1,282 @@
+require 'hammer_cli_foreman/image'
+require 'hammer_cli_foreman/compute_resources'
+
+module HammerCLIForeman
+  class Attribute < HammerCLIForeman::Command
+    resource :compute_attributes
+
+    def new_element_id
+      Time.now.to_i.to_s
+    end
+
+    def self.get_params(options)
+      params = {}
+      params['compute_attribute'] = {}
+      profile = HammerCLIForeman.record_to_common_format(
+          HammerCLIForeman.foreman_resource(:compute_profiles).call(:show, 'id' => options['option_compute_profile_id'] )
+      )
+      HammerCLIForeman.foreman_resource(:compute_resources).call(:show, 'id' => options['option_compute_resource_id'] )
+      params['compute_attribute'] = profile['compute_attributes'].select { |hash| hash['compute_resource_id'] == options['option_compute_resource_id']}[0] || {}
+      params
```

Comment:
I think we need to remove the 'attributes' which is read-only (?) copy of the compute_attributes. 
```ruby
params["compute_attribute"].delete("attributes")
```